### PR TITLE
[#1717] fix(jdbc-mysql): throw an exception if setting comment on MySQL database

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
throw a exception when user. set comment on MySQL database

### Why are the changes needed?

Fix: #1717 #1767 

### Does this PR introduce _any_ user-facing change?
yes, if user  set comment on MySQL database, will throw a exception.

### How was this patch tested?
add UT

### CODE REVIEW

File: catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java [hunk 1]
```diff
@@ -39,9 +40,10 @@ public class MysqlDatabaseOperations extends JdbcDatabaseOperations {
   @Override
   public String generateCreateDatabaseSql(
       String databaseName, String comment, Map<String, String> properties) {
-    if (StringUtils.isNotEmpty(comment)) {
-      LOG.warn(
-          "Ignoring comment option on database create. mysql does not support comment option on database create.");
+    String originComment = StringIdentifier.removeIdFromComment(comment);
+    if (StringUtils.isNotEmpty(originComment)) {
+      throw new UnsupportedOperationException(
+          "Mysql doesn't support set schema comment:" + originComment);
```
Comment (yuqi1129) [inline catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java:46]: MySQL

Comment (FANNG1) [reply catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java:46]: ok

Comment (FANNG1) [reply catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java]: fixed

File: catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java [hunk 2]
```diff
@@ -39,9 +40,10 @@ public class MysqlDatabaseOperations extends JdbcDatabaseOperations {
   @Override
   public String generateCreateDatabaseSql(
       String databaseName, String comment, Map<String, String> properties) {
-    if (StringUtils.isNotEmpty(comment)) {
-      LOG.warn(
-          "Ignoring comment option on database create. mysql does not support comment option on database create.");
+    String originComment = StringIdentifier.removeIdFromComment(comment);
+    if (StringUtils.isNotEmpty(originComment)) {
+      throw new UnsupportedOperationException(
+          "MySQL doesn't support set schema comment:" + originComment);
```
[the remark this answers is not in the document]
Comment (FANNG1) [reply catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java:46]: ok

**Review by jerryshao — APPROVED** [review]

**Pull request merged**